build/unix/module.modulemap

```diff
@@ -14,7 +14,7 @@ module ROOT_Types {
   // FIXME: This module should contain only header files with types.
   // The listed below headers are here because we want them to work in both
   // -frtti and -fno-rtti environment. Move them into a separate module.
-  module "RStringView.h" { header "RStringView.h" export * }
+  module "RStringView.h" { header "RStringView.h" export * } // TODO substitute with ROOT/RStringView.hxx
```

Comment:
Putting `header "ROOT/RStringView.cxx"` should suffice.

Reply:
ok so I should leave `module "RStringView.h"` as it is?

Reply:
Well, as you prefer. The module name is just an identifier. Maybe for consistency we should prefix it with ROOT/, too.

Reply:
alright thanks, done